llvm/test/MC/AMDGPU/gfx12_asm_vop3_from_vop1_dpp16.s

```diff
@@ -1932,48 +1932,6 @@ v_log_f32_e64_dpp v5, v1 mul:4 row_xmask:0 row_mask:0x1 bank_mask:0x3 bound_ctrl
 v_log_f32_e64_dpp v255, -|v255| clamp div:2 row_xmask:15 row_mask:0x3 bank_mask:0x0 bound_ctrl:0 fi:1
 // GFX12: [0xff,0x81,0xa7,0xd5,0xfa,0x00,0x00,0x38,0xff,0x6f,0x05,0x30]
 
-v_mov_b32_e64_dpp v5, v1 quad_perm:[3,2,1,0]
```

Comment:
Why have you removed these v_mov_b32_e64_dpp tests here and in the other files? Was it a scripting mistake? They seem to be testing valid cases.